src/test/java/org/apache/commons/lang3/StringUtilsTest.java

```diff
@@ -3301,4 +3301,22 @@ public void testToRootUpperCase() {
             Locale.setDefault(defaultLocales);
         }
     }
+
+    @Test
+    public void testGeorgianSample() {
+        char[] arrayI = new char[]{(char) 305, (char) 1012};
```

Comment:
Please use hexadecimal for the values with a comment that shows the Unicode name for each character. Otherwise, there is not way to tell what this is doing without knowing the Unicode values from memory.

Reply:
@garydgregory done.